HEPML.bib

```diff
@@ -1,5 +1,18 @@
 # HEPML Papers
 
+%May 30, 2023
+@article{ATLAS:2023mny,
+    author = "Aad, Georges and others",
+    collaboration = "ATLAS",
```

Comment:
While this is what HEPData spit out, this kinda always looks terrible (as HEPData has been told many times) and I've taken to just replacing it with 
```suggestion
    author = "{ATLAS Collaboration}",
```

Reply:
> This is inconsistent with many of the other entries in the review 

Yeah there are a lot that were just added without checks.

> doesn't follow the instructions of the CONTRIBUTING.md

Good point. Made Issue #151 for this.

> I think it's better to add this one this way, and then to edit everything consistently in a new PR. Let me know if you agree, for now I've taken no action.

That's a valid concern. It is going to get changed though so I'll just commit the suggested change and move this forward. No further action needed on your end and thanks for the PR!